<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>display</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="idInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="expiryInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="typeInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Preferred</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name          :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID               :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Expire date  : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>update </string>
    </property>
   </widget>
   <widget class="QPushButton" name="rebateButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>rebate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadSalesFile">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Sales File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateReport">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Report</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reportDayInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="reportDayLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="reportMonthLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reportMonthInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="reportYearLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="reportYearInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="itemsSoldReport">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Items Sold Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allPurchaseReport">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User's All Purchase Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="totalPurchaseReport">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Purchases by All Members</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="reportWindow">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>411</width>
      <height>561</height>
     </rect>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="membershipDueReport">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Yearly Membership Dues Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="membersUpgrade">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Basic Members Who Should Upgrade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="membersDowngrade">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Preferred Members Who Should Downgrade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemNameInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>398</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="itemNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="itemUnitsSold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Total Units Sold For Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
